docs/manage/data-deletion.md

```diff
@@ -0,0 +1,79 @@
+---
+id: data-deletion
+title: Data Deletion Requests
+description: Learn how to independently and efficiently control or remove your sensitive data from Sumo Logic.
+---
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Data Deletion allows you to quickly and easily request the removal of ingested data from Sumo Logic. This is particularly useful for removing inadvertently ingested sensitive data.
+
+You can independently and swiftly remove sensitive data, mitigating compliance issues and potential data exposure without needing to contact Sumo Logic support.
+
+Key features:
+
+- **User-friendly deletion**. Easily delete data.
+- **Multiple datasets and time ranges**. Flexible data management.
+- **Customizable filters**. Tailor deletion to your needs.
+- **Robust auditing mechanisms**. Ensure thorough tracking.
+
+
+## Create a Data Deletion Request
+
+:::warning
+Deletion requests result in the permanent deletion of data. Once deleted, the data cannot be recovered. Ensure that you have appropriately backed up any necessary data before submitting a deletion request.
+:::
```

Comment:
@krastogi-eng , I created a warning message about permanent deletion, and moved it higher up in the doc. Does this suffice?

Reply:
yes, this looks good. Sorry i missed it earlier

Reply:
perfect! @krastogi-eng thanks for reviewing.